vnext/ReactWindowsCore/IWebSocket.h

```diff
@@ -73,41 +81,115 @@ struct IWebSocket
 
   #pragma endregion // Inner types
 
+  /// <summary>
+  /// Creates an <c>IWebSocket</c> instance.
+  /// </summary>
+  /// <param name="url">
+  /// WebSocket URL address the instance will connect to.
+  /// The address's scheme can be either ws:// or wss://.
+  /// </param>
   static std::unique_ptr<IWebSocket> Make(const std::string& url);
 
   // Only use if a legacy implementation is required as fallback.
```

Comment:
>// Only use if a legacy implementation is required as fallback. [](start = 2, length = 63)

Is this still relevant ? What does legacy implementation mean here ?

Reply:
The `Legacy` implementation blocks the main thread during connection.
We kept it to deploy the updated (non-blocking) functionality behind a feature gate.
Once the feature gate is removed, so will it be for `LegacyWebSocket.cpp`.